lms/templates/report_modal.html

```diff
@@ -105,7 +106,7 @@
     });
     $("#report-form").submit(function() {
         $("input[type='submit']", this).attr("disabled", "disabled");
-        $('area,input,select,textarea,button').attr('tabindex', -1);
+        $('area,input,select,textarea,button', '#report-form-wrapper').attr('tabindex', -1);
```

Comment:
this is incorrect, it should be: $('area,input,select,text area,button,#report-form-wrapper').attr('tabindex', -1);